bots/validatespeakerscornerissue.py

```diff
@@ -0,0 +1,118 @@
+import os
+from datetime import date, timedelta
+from github import Github
+import arxiv
+
+VSF_BOT_TOKEN = os.getenv("VSF_BOT_TOKEN")
+ISSUENUMBER = os.getenv("ISSUENUMBER")
+
+g = Github(VSF_BOT_TOKEN)
+repo = g.get_repo("virtualscienceforum/virtualscienceforum")
+issue = repo.get_issue(number=ISSUENUMBER)
+
+def CheckDate(dateStr, timeStr):
+    scheduledDate = date.fromisoformat(dateStr)
```

Comment:
`YYYY-MM-DD HH-MM UTC` isn't isoformat, we should think how to do this better. dateutil may help.

Reply:
The variable `dateStr` is only the `YYYY-MM-DD` bit. We can be more strict and use the full datetime and use 14*24 hrs for the threshold.

Reply:
Ah, nvm then, should be fine. But we will want to parse the full thing regardless.

Reply:
@akhmerov would you mind having a go at this?

Reply:
Sure, I'll have some time over the weekend.